core/configuration/motd

```diff
@@ -0,0 +1,16 @@
+
+Welcome to Companion 1.0!
+
+What you currently see is a tmux session built into Companion docker.
+This means most of the data in here is not-persistent, except for things in /root/.config
```

Comment:
It's better to just say that nothing is persistent and do not tell about /root/.config, otherwise people will start using it to do sutff